README.md

```diff
@@ -368,25 +368,31 @@ This method will remove profiling information that was related to previous page
 You need to inject the following in your SPA to load MiniProfiler's speed badge ([extra details surrounding this script](https://github.com/MiniProfiler/rack-mini-profiler/issues/139#issuecomment-192880706) and [credit for the script tag](https://github.com/MiniProfiler/rack-mini-profiler/issues/479#issue-782488320) to [@ivanyv](https://github.com/ivanyv)):
 
 ```html
- <script type="text/javascript" id="mini-profiler"
-        src="/mini-profiler-resources/includes.js?v=12b4b45a3c42e6e15503d7a03810ff33"
-        data-css-url="/mini-profiler-resources/includes.css?v=12b4b45a3c42e6e15503d7a03810ff33"
-        data-version="12b4b45a3c42e6e15503d7a03810ff33"
-        data-path="/mini-profiler-resources/"
-        data-horizontal-position="left"
-        data-vertical-position="top"
-        data-ids=""
-        data-trivial="false"
-        data-children="false"
-        data-max-traces="20"
-        data-controls="false"
-        data-total-sql-count="false"
-        data-authorized="true"
-        data-toggle-shortcut="alt+p"
-        data-start-hidden="false"
-        data-collapse-results="true"
-        data-html-container="body"
-        data-hidden-custom-fields></script>
+<script
+      type="text/javascript"
```

Comment:
thank you for your comment but its the same as in the old script in the readme file and i think if you remove it, it will create the  MIME type error in the browsers.

also about the data-ids, when i leave it empty it doesnt work, how can i dynamically catch the correct id and place it in it?